tests/schema/test_lazy_types/test_cyclic.py

```diff
@@ -30,5 +31,23 @@ class Query:
     """
 
     schema = strawberry.Schema(Query)
-
-    assert print_schema(schema) == textwrap.dedent(expected).strip()
+    res = schema.execute_sync(
+        """
+    query MyQuery {
+      a {
+        typeB {
+          typeA {
+            done
+          }
+        }
+      }
+    }
+    """,
+        root_value=Query(a=TypeA(), b=TypeB()),
+    )
+    assert res.errors is None
+    assert res.data["a"]["typeB"]["typeA"]["done"]
+    expected = textwrap.dedent(expected).strip()
+    schema = print_schema(schema).splitlines()
+    for line in expected.splitlines():
+        assert line in schema
```

Comment:
Why this change? Are we not guaranteeing order?

Reply:
See above

Reply:
Is order mandatory?